Author: David Martinez Marti <deavidsedice@gmail.com>
License: GNU GPL, see /usr/share/common-licenses/GPL
Description: Ensure that the sound does not disappear completely when you fail a note and there's no base soundtrack
Last-Update: 2008-07-23
--- a/src/Song.py
+++ b/src/Song.py
@@ -525,6 +525,8 @@
     self.engine.audio.unpause()
 
   def setGuitarVolume(self, volume):
+    if not self.rhythmTrack:
+        volume+=0.1
     if self.guitarTrack:
       self.guitarTrack.setVolume(volume)
     else:
